src/smolagents/models.py

```diff
@@ -447,43 +480,43 @@ def __init__(
         if not _is_package_available("vllm"):
             raise ModuleNotFoundError("Please install 'vllm' extra to use VLLMModel: `pip install 'smolagents[vllm]'`")
 
-        from vllm import LLM
-        from vllm.transformers_utils.tokenizer import get_tokenizer
+        from vllm import LLM  # type: ignore
+        from vllm.transformers_utils.tokenizer import get_tokenizer  # type: ignore
 
-        self.model_kwargs = {
-            **(model_kwargs or {}),
-            "model": model_id,
-        }
+        self.model_kwargs = model_kwargs or {}
         super().__init__(**kwargs)
         self.model_id = model_id
-        self.model = LLM(**self.model_kwargs)
+        self.model = LLM(model=model_id, **self.model_kwargs)
+        assert self.model is not None
```

Comment:
I think these changes are not related to streaming. But is it necessary the assert here? I mean, any model is prone to receiving a `None` as `model_id`...